its/ruling/src/test/resources/sonar-server/squid-S3010.json

```diff
@@ -0,0 +1,5 @@
+{
+'org.codehaus.sonar:sonar-server:src/main/java/org/sonar/server/platform/MasterServletFilter.java':[
+48,
```

Comment:
wonder if the rule should be smart... and identify singleton patterns (by maybe ignoring fields called `singleton`, `instance` or `uniqueInstance`).